.ci_scripts/beforeInstall.sh

```diff
@@ -30,21 +32,25 @@ fi
 
 echo "Max connection: $MAX_CON"
 
-# OpenSSL
```

Comment:
Sure I can revert it, it's just the compilation takes some time and it slows down the suites that don't need it.